Dockerfile

```diff
@@ -138,6 +138,41 @@ RUN apt-get update \
   && pip3 install protoc-docs-plugin \
   && rm -rf /var/lib/apt/lists/*
 
+# Install .NET Core SDK
+# Install .NET CLI dependencies
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+    libc6 \
+    libcurl3 \
+    libgcc1 \
+    libgssapi-krb5-2 \
+    liblttng-ust0 \
+    libssl1.0.0 \
+    libstdc++6 \
+    libunwind8 \
+    libuuid1 \
+    zlib1g \
+    && rm -rf /var/lib/apt/lists/*
+
+# Install .NET Core SDK
+ENV DOTNET_SDK_VERSION 1.0.4
+ENV DOTNET_SDK_DOWNLOAD_URL https://dotnetcli.blob.core.windows.net/dotnet/Sdk/$DOTNET_SDK_VERSION/dotnet-dev-ubuntu.16.04-x64.$DOTNET_SDK_VERSION.tar.gz
+
+RUN curl -SL $DOTNET_SDK_DOWNLOAD_URL --output dotnet.tar.gz \
+    && mkdir -p /usr/share/dotnet \
+    && tar -zxf dotnet.tar.gz -C /usr/share/dotnet \
+    && rm dotnet.tar.gz \
+    && ln -s /usr/share/dotnet/dotnet /usr/bin/dotnet
+
+# Trigger the population of the local package cache
+ENV NUGET_XMLDOC_MODE skip
```

Comment:
Unless it will be drastically slower, I would favor the space than time spent on generation. But if it will only save couple of MBs, let's keep it.